### ISSUE TITLE
Web: Support `Label` tag without `forId`

### ISSUE THREAD

**Issue by SebastianAigner** [JetBrains/compose-jb#640]
The `Label` tag has a parameter `forId`, which is used to associate a label with an input, i.e. corresponding to this HTML style:

```
<div class="preference">
    <label for="peas">Do you like peas?</label>
    <input type="checkbox" name="peas" id="peas">
</div>
```

However, the `input` can also be nested inside the `label`, which then doesn't need to have a `forId`:

```
<label>Do you like peas?
  <input type="checkbox" name="peas">
</label>
```

(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

So `Label`'s `forId` should probably be nullable.

**Comment by eymar** [comment]
We published a new version - `0.0.0-web-dev-12`. It contains the fix.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6427">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
